src/room_member.cpp

```diff
@@ -175,8 +175,7 @@ void RoomMember::ReceiveLoop() {
 };
 
 void RoomMember::Join(const std::string& nickname, const std::string& server, uint16_t server_port, uint16_t client_port) {
-    // TODO(Subv): Use client_port.
-    RakNet::SocketDescriptor socket;
+    RakNet::SocketDescriptor socket(client_port, 0);
```

Comment:
So internally RakNet checks if (hostAddress)  /* With host address the second argument */
if false, it will assign hostAddress[0] = 0. Thus if hostAddress=nullptr this will fail...

We should stay with 0